workspaces/Makefile

```diff
@@ -0,0 +1,27 @@
+shared:
```

Comment:
Do you want `shared` to be the default target here?

Reply:
I put an empty `default` target